rotkehlchen/externalapis/etherscan.py

```diff
@@ -285,11 +289,56 @@ def get_transactions(
             for entry in result:
                 gevent.sleep(0)
                 try:
-                    tx = deserialize_ethereum_transaction(  # type: ignore
-                        data=entry,
-                        internal=is_internal,
-                        ethereum=None,
-                    )
+                    # Handle genesis block transactions
+                    if is_genesis_tx(entry) is False:
+                        tx = deserialize_ethereum_transaction(  # type: ignore
+                            data=entry,
+                            internal=is_internal,
+                            ethereum=None,
+                        )
+                    else:
+                        # Handling genesis transactions
+                        if self.db is None:
+                            raise DeserializationError('DB not initialized')
+                        dbtx = DBEthTx(self.db)
+                        tx_in_db = dbtx.get_ethereum_transactions(
+                            filter_=ETHTransactionsFilterQuery.make(tx_hash=GENESIS_HASH),
+                            has_premium=True,
+                        )
+                        if len(tx_in_db) == 1:
+                            tx = tx_in_db[0]
+                        else:
+                            tx = EthereumTransaction(
+                                timestamp=ETHEREUM_BEGIN,
+                                block_number=0,
+                                tx_hash=GENESIS_HASH,
+                                from_address=ZERO_ADDRESS,
+                                to_address=None,
+                                value=0,
+                                gas=0,
+                                gas_price=0,
+                                gas_used=0,
+                                input_data=b'',
+                                nonce=0,
+                            )
+                            dbtx.add_ethereum_transactions(
+                                ethereum_transactions=[tx],
+                                relevant_address=account,
+                            )
+                            trace_id = dbtx.get_max_trace_id(GENESIS_HASH)
+                            trace_id = trace_id + 1 if trace_id is not None else 0
+                            entry['from'] = ZERO_ADDRESS
+                            entry['hash'] = GENESIS_HASH
+                            entry['traceId'] = trace_id
+                            internal_tx = deserialize_ethereum_transaction(
+                                data=entry,
+                                internal=True,
+                                ethereum=None,
+                            )
+                            dbtx.add_ethereum_internal_transactions(
+                                transactions=[internal_tx],
+                                relevant_address=account,
+                            )
```

Comment:
```suggestion
                            trace_id = dbtx.get_max_trace_id(GENESIS_HASH)
                            trace_id = trace_id + 1 if trace_id is not None else 0
                            entry['from'] = ZERO_ADDRESS
                            entry['hash'] = GENESIS_HASH
                            entry['traceId'] = trace_id
                            internal_tx = deserialize_ethereum_transaction(
                                data=entry,
                                internal=True,
                                ethereum=None,
                            )
                            dbtx.add_ethereum_internal_transactions(
                                transactions=[internal_tx],
                                relevant_address=account,
                            )
```

This is indented wrong. These should all be on level of indentation to the left since the if/else should basically create or return the genesis ethereum transaction.

Reply:
Yeah, thanks